flipperdevices/Flipper-Android-App
# Add screen with permission request

**Background**

Before start connection we need check all granted permission

**Changes**

- Add permission screen and dispatch edge cases
- Add routing library

**Test plan**

Test case 01:
- Open application without bluetooth enabled
- Click to button
- Check that application suggest enabled bluetooth

Test case 02:
- Open application without permission granted but with bluetooth enabled
- Click to button
- Check that application suggest grant permission

Test case 03:
- Open application with all permission granted
- Click to button
- Check that application write "Yay!"
